hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/lib/output/FileOutputCommitter.java

```diff
@@ -348,6 +348,16 @@ public Path getWorkPath() throws IOException {
    * @param context the job's context
    */
   public void setupJob(JobContext context) throws IOException {
+    // Downgrade v2 to v1 with a warning.
+    if (algorithmVersion == 2) {
+      Logger log = LoggerFactory.getLogger(
+          "org.apache.hadoop.mapreduce.lib.output."
+              + "FileOutputCommitter.Algorithm");
+
+      log.warn("The v2 commit algorithm is deprecated;"
+          + " please switch to the v1 algorithm");
```

Comment:
I don't think we should use the word deprecated.   That implies that this algorithm will be removed in a future release

Reply:
what do you suggest?

Reply:
switching to your text